specs/executionAPISpec.md

```diff
@@ -0,0 +1,530 @@
+## <p id="maxPriorityFeePerGas">eth_maxPriorityFeePerGas</p>
+* [EMPFPG-1] eth_maxPriorityFeePerGas **MUST** return the clients price per unit of gas - 7.
+## <p id="feeHistory">eth_feeHistory</p>
+* [EFH-1] eth_feeHistory **MUST** return the following information for number of blocks specified by the `blockCount` parameter stopping at the `highestBlock` parameter.
+  * An array containing the base fee per gas for each block plus then block after `highestBlock`.
+  * An array containing the ratio of the gas used by each block.
+  * An array containing arrays with the requested `rewardPercentiles` for each block.
+  * The oldest block used for the request.
+* [EFH-2] eth_feeHistory **MUST** use the available range of blocks when the requested `blockCount` range can't be retrieved.
+* [EFH-3] eth_feeHistory **MUST** allow block tags to be used for `highestBlock`.
+* [EFH-4] eth_feeHistory **MUST** error with code -32000 when `highestBlock` is ahead of the chain.
+## <p id="blockNumber">eth_blockNumber</p>
+* [EBN-1] eth_blockNumber **MUST** return the block number for the most recent block mined.
+* [EBN-2] eth_blockNumber **MUST** return "0x0" when the client is not synced to the network.
+## <p id="sign">eth_sign</p>
+* [ESN-1] eth_sign **MUST** return the Ethereum specific signature detailed in [EIP-191](https://eips.ethereum.org/EIPS/eip-191) for the given unlocked `address` and `message`.
+* [ESN-2] eth_sign **MUST** error with code -32000 when the account corresponding to the `address` is not unlocked.
+* [ESN-3] eth_sign **MUST** error with code -32000 when the account corresponding to the `address` is not owned by the client.
+## <p id="call">eth_call</p>
+* [EC-1] eth_call **MUST** return the result of the given transaction.
+* [EC-2] eth_call **MUST** accept all current transaction types. Legacy transactions and [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) "typed" transactions.
+* [EC-3] eth_call **MUST NOT** mine any transaction on the blockchain.
+* [EC-4] eth_call **MUST** use the block requested by the `defaultBlockParameter` when interacting with contracts.
+* [EC-5] eth_call **MUST** error with code -32000 when the requested block does not exist or is not available.
+* [EC-6] eth_call **SHOULD NOT** be allowed to be called from an address where CODEHASH != EMPTYCODEHASH. [EIP-3607](https://eips.ethereum.org/EIPS/eip-3607)
+* [EC-7] eth_call **MUST NOT** error if the transaction exceeds the gas or fee cap.
+* [EC-8] eth_call **MUST** use 0x0000000000000000000000000000000000000000 as default `from` address when `from` is null or not specified. [Nethermind uses 0xf...fe] 
+* [EC-9] eth_call **MUST** check `from` account balance has sufficient funds to "pay" for the transaction.
+* [EC-10] eth_call **MUST** error with code -32000 account has insufficient funds to execute the transaction call.
+* [EC-11] eth_call **MUST NOT** calculate cost of transactions when the `gas` and a price per unit of gas is not specified.   
```

Comment:
What does this mean?  `eth_call` doesn't return gas usage numbers currently.  It seems like an implementation detail whether gas account is tracked since it isn't exposed at all.